vaadin-platform-test/src/main/resources/vaadin-featureflags.properties

```diff
@@ -0,0 +1,2 @@
+# Map component (Pro)
+com.vaadin.experimental.mapComponent=true
```

Comment:
I'm doing some modifications in flow so as we don't need to enable experimental components in our test suite.
Blocking this for now

Reply:
ok lets merge this, and fix flow later